packages/gatsby-remark-copy-linked-files/README.md

```diff
@@ -15,7 +15,12 @@ plugins: [
     resolve: `gatsby-transformer-remark`,
     options: {
       plugins: [
-        `gatsby-remark-copy-linked-files`,
+        { 
+          resolve: 'gatsby-remark-copy-linked-files',
+          options: {
+            ignoreFileExtensions: ['png', 'jpg', 'jpeg', 'bmp', 'tiff'],
```

Comment:
So you don't need to add it.

Or even better, show the simple default way to add the plugin to gatsby-config and what to do if you want to copy images over (e.g. an empty `ignoreFileExtensions`.